README.md

```diff
@@ -78,9 +78,27 @@ When a change is committed, a Jenkins job will recompile and re-deploy mattermod
 
 In order to test Mattermod locally a couple of steps are needed.
 
-* 1\. Local running version of MYSQL database. You can easily spin up a database by executing ```docker run --name mattermod-mysql -p 3306:3306 -e MYSQL_ROOT_PASSWORD=mattermod -e MYSQL_USER=mattermod -e MYSQL_PASSWORD=mattermod -e MYSQL_DATABASE=mattermod -d mysql:5.7 > /dev/null;```
```

Comment:
I think we should leave in the part about requiring MySQL and maybe add a note about updating the dsn in the config file.

Reply:
Hehe.. was going to mention about the DSN. Sure, will add the MySQL part.